Makefile

```
@@ -0,0 +1,22 @@
+.PHONY: test test-unit test-coverage test-watch setup-test-env
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)